package/doc/sphinx/source/documentation_pages/references.rst

```diff
@@ -65,170 +51,182 @@ Citations for included algorithms and modules
+   :labelprefix: ᵃ
+
+   ArayaSecchi2014
 
-.. [Araya-Secchi2014b] R. Araya-Secchi., Tomas Perez-Acle, Seung-gu Kang, Tien
-   Huynh, Alejandro Bernardin, Yerko Escalona, Jose-Antonio Garate,
-   Agustin D. Martinez, Isaac E. Garcia, Juan C. Saez, Ruhong
-   Zhou. Characterization of a novel water pocket inside the human Cx26
-   hemichannel structure. *Biophysical Journal*, **107** (2014), 599-612.
 
 If you use the Path Similarity Analysis (PSA) code in
-:mod:`MDAnalysis.analysis.psa` please cite [Seyler2015b]_.
+:mod:`MDAnalysis.analysis.psa` please :cite:p:`a-Seyler2015`.
 
-.. [Seyler2015b] Seyler SL, Kumar A, Thorpe MF, Beckstein O. Path Similarity
-  Analysis: A Method for Quantifying Macromolecular Pathways. *PLoS
-  Comput Biol* **11** (2015), e1004568. doi: `10.1371/journal.pcbi.1004568`_
+.. bibliography::
+   :filter: False
+   :style: MDA
+   :keyprefix: a-
+   :labelprefix: ᵃ
 
-.. _`10.1371/journal.pcbi.1004568`: http://doi.org/10.1371/journal.pcbi.1004568
+   Seyler2015
 
 If you use the implementation of the ENCORE ensemble analysis in
-:mod:`MDAnalysis.analysis.encore` please cite [Tiberti2015b]_.
+:mod:`MDAnalysis.analysis.encore` please cite :cite:p:`a-Tiberti2015`.
 
-.. [Tiberti2015b] M. Tiberti, E. Papaleo, T. Bengtsen, W. Boomsma,
-   and K. Lindorff-Larsen. ENCORE: Software for quantitative ensemble
-   comparison. *PLoS Comput Biol*, **11** (2015), e1004415.  doi:
-   `10.1371/journal.pcbi.1004415`_
+.. bibliography::
+   :filter: False
+   :style: MDA
+   :keyprefix: a-
+   :labelprefix: ᵃ
 
-.. _`10.1371/journal.pcbi.1004415`: http://doi.org/10.1371/journal.pcbi.1004415
+   Tiberti2015
 
 If you use the streamline visualization in
 :mod:`MDAnalysis.visualization.streamlines` and
-:mod:`MDAnalysis.visualization.streamlines_3D` please cite [Chavent2014b]_.
+:mod:`MDAnalysis.visualization.streamlines_3D` please cite :cite:p:`a-Chavent2014`.
 
-.. [Chavent2014b] Chavent, M., Reddy, T., Dahl, C.E., Goose, J., Jobard, B.,
-   and Sansom, M.S.P. Methodologies for the analysis of instantaneous lipid
-   diffusion in MD simulations of large membrane systems.  *Faraday
-   Discussions* **169** (2014), 455–475. doi: `10.1039/c3fd00145h`_
+.. bibliography::
+   :filter: False
+   :style: MDA
+   :keyprefix: a-
+   :labelprefix: ᵃ
 
-.. _`10.1039/c3fd00145h`: https://doi.org/10.1039/c3fd00145h
+   Chavent2014
 
 If you use the hydrogen bond analysis code in
-:mod:`MDAnalysis.analysis.hydrogenbonds.hbond_analysis` please cite [Smith2019]_.
+:mod:`MDAnalysis.analysis.hydrogenbonds.hbond_analysis` please cite :cite:p:`a-Smith2019`.
 
-.. [Smith2019] P. Smith, R. M. Ziolek, E. Gazzarrini, D. M. Owen, and C. D. Lorenz.
-   On the interaction of hyaluronic acid with synovial fluid lipid membranes. *PCCP*
-   **21** (2019), 9845-9857. doi:  `10.1039/C9CP01532A`_
+.. bibliography::
+   :filter: False
+   :style: MDA
+   :keyprefix: a-
+   :labelprefix: ᵃ
 
-.. _`10.1039/C9CP01532A`: http://dx.doi.org/10.1039/C9CP01532A
+   Smith2019
```

Comment:
This reference appears on the `references.rst` page but not in the docstrings of `MDAnalysis.analysis.hydrogenbonds.hbond_analysis`. Please let me know if the preference is to add it to the analysis file as well.

Reply:
Yes, I think it should be in the analysis file, too — @p-j-smith ?

Reply:
Yes please!